styles.css

```diff
@@ -0,0 +1,335 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: white;
+  font-family: 'Karla', sans-serif;
+}
+
+h1 {
+  color: #2B547E;
+}
+
+h2 {
+  color: #2B547E;
+
+}
+
+.header-container {
+  background-image: url("images/header-image.jpg");
+  background-repeat: no-repeat;
+  display: flex;
+  background-color: grey;
+  width: 100%;
+  height: 30rem;
+  position: relative;
+}
+
+.header-logo {
```

Comment:
if you add flex direction column to this class it might help the image and text lay out better.
Also, the width is kind of skinny, making the text cut off too soon, maybe consider making this wider?